types/bip21/index.d.ts

```diff
@@ -1,11 +1,11 @@
-// Type definitions for bip21 v1.1.2
+// Type definitions for bip21 v2.0.0
 // Project: https://github.com/bitcoinjs/bip21
 // Definitions by: Stefan Huber <https://github.com/stefanhuber>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare namespace bip21 {
-    export function decode(uri:string) : {address:string,amount?:number | undefined};
-    export function encode(address:string,options?:any) : string;
+    export function decode(uri: string): { address: string; amount?: number | undefined; options: object | undefined };
```

Comment:
this looks wrong, IMO only `address` and `amount` are returned in the object:
https://github.com/bitcoinjs/bip21/blob/master/index.js#L24